Bottom: a956c1bf43e3bb9be12ab7782b3981886ab2b254
Top:    5034802b08a8450a8bb3975d63e6de0b427bf598
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 16:34:05 +0100

Refresh of hid-gerber-don-t-use-the-core-

---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index e1f139c..4518b21 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -466,35 +466,6 @@ assign_file_suffix (char *dest, int idx)
   strcat (dest, sext);
 }
 
-typedef struct
-{
-  int nplated;
-  int nunplated;
-} HoleCountStruct;
-
-static int
-hole_counting_callback (const BoxType * b, void *cl)
-{
-  PinTypePtr pin = (PinTypePtr) b;
-  HoleCountStruct *hcs = cl;
-  if (TEST_FLAG (HOLEFLAG, pin))
-    hcs->nunplated++;
-  else
-    hcs->nplated++;
-  return 1;
-}
-
-static void
-count_holes (BoxType *region, int *plated, int *unplated)
-{
-  HoleCountStruct hcs;
-  hcs.nplated = hcs.nunplated = 0;
-  r_search (PCB->Data->pin_tree, region, NULL, hole_counting_callback, &hcs);
-  r_search (PCB->Data->via_tree, region, NULL, hole_counting_callback, &hcs);
-  if (plated != NULL) *plated = hcs.nplated;
-  if (unplated != NULL) *unplated = hcs.nunplated;
-}
-
 static int
 hole_callback (const BoxType * b, void *cl)
 {
